dispatch/apps/events/views.py

```diff
@@ -32,6 +35,16 @@ def submit_form(request):
             form.is_submission = True
             form.save()
 
+            template = render_to_string('events/email/confirm.html', form.cleaned_data)
+
+            send_mail(
+                    'Your event has been created!',
+                    template,
+                    settings.EMAIL_HOST_USER,
+                    [form.cleaned_data['submitter_email']],
+                    fail_silently=True,
+                )
```

Comment:
This just occurred to me: for consistency with the published email, do you think it would make sense to send this when the `post_create` signal is fired?